.github/workflows/ci.yml

```diff
@@ -221,47 +212,51 @@ jobs:
 
     strategy:
       matrix:
-        compiler: ["gcc-13", "clang-18"]
+        os:
+          - "ubuntu-latest"
```

Comment:
i Think we should versionize this. Not going to be a problem in the next two years, but still